test/is-required-flag.js

```diff
@@ -1,7 +1,9 @@
 import test from 'ava';
 import execa from 'execa';
-const path = require('path');
+import path from 'path';
```

Comment:
```suggestion
import path from 'node:path';
```

For all the built-in modules. https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c